detail.html

```diff
@@ -0,0 +1,34 @@
+<%inherit file="app_base.htm" />
+
+<%block name ="content_top">
+</%block>
```

Comment:
This block seems unnecessary. It has no content within.

Reply:
that is true my dude